include/dungeon_global_data.h

```diff
@@ -197,7 +198,9 @@ struct Dungeon
     /* 0x17B34 */ u8 fillunk1734[0x17B38 - 0x17B34];
     /* 0x17B38 */ u32 unk17B38;
     /* 0x17B3C */ u32 unk17B3C;
-    u8 fill17B40[0x181E8 - 0x17B40];
+    u8 fill17B40[0x17B44 - 0x17B40];
+    struct Dungeon_Sub17B44 *sub17B44[MONSTER_MAX]; // Assuming MONSTER_MAX since the highest value I've seen read is MONSTER_DECOY
```

Comment:
Should be 0x1A8 -> 424 so `MONSTER_RAYQUAZA_CUTSCENE` + 1 based off of 0x80688CC. On my branch, I've already decomped it as:
```c
void CloseAllSpriteFiles(void)
{
    s32 index;

    for(index = 0; index < 0x1A8; index++)
    {
        if (gDungeon->sprites[index] != NULL) {
            CloseFile(gDungeon->sprites[index]);
            gDungeon->sprites[index] = NULL;
        }
    } 
}

```